activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -382,7 +382,7 @@ def rename_table(table_name, new_name)
           execute "ALTER TABLE #{quote_table_name(table_name)} RENAME TO #{quote_table_name(new_name)}"
           pk, seq = pk_and_sequence_for(new_name)
           if pk
-            idx = "#{table_name}_pkey"
+            idx = "#{table_name[0, index_name_length - '_pkey'.size]}_pkey"
```

Comment:
It's a nitpick and totally up to you but I really find the `"#{table_name}_pkey"` very readable and was wondering if we could somehow keep it, something like:
```suggestion
            table_name_max_length =  index_name_length - '_pkey'.size
            truncated_table_name = table_name[0,table_name_max_length]
            idx = "#{truncated_table_name}_pkey"
```
Not sure about the `truncated_table_name` name though as it's not always truncated, perhaps `limited_table_name`?